iidm/iidm-extensions/src/main/java/com/powsybl/iidm/network/extensions/VoltagePerReactivePowerControl.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.extensions;
+
+import com.powsybl.commons.extensions.Extension;
+import com.powsybl.iidm.network.StaticVarCompensator;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
```

Comment:
Fix the author (same remark for all the files)

Reply:
Indeed it is from @geofjamg I just have moved some code.